build.gradle

```diff
@@ -175,8 +175,13 @@ subprojects {
             cacheChangingModulesFor 5, 'days'
         }
 
-        exclude(group: "ch.qos.logback", module: "logback-core")
-        exclude(group: "ch.qos.logback", module: "logback-classic")
+        if (!project.name.contains('cassandra-ticket-registry')) {
```

Comment:
Don't think so... I submitted a patch to cassandra-unit, the embedded cassandra project, not to the official cassandra DB project, where the problem is as they expect a logback logger.

In my opinion my solution was fine, exclude logback for all the subprojects except cassandra that needs it. Actually I would like to exclude it only for testCompile configuration